blob/s3blob/s3blob.go

```diff
@@ -14,6 +14,12 @@
 
 // Package s3blob provides an implementation of blob using S3.
 //
+// For blob.Open URLs, s3blob registers for the "s3" protocol.
+// The URL's Host is used as the bucket name.
+// The following query options are supported:
+// - region: The AWS region for requests.
```

Comment:
Where does the auth go?

Reply:
I updated the comment and added a link to the [AWS documentation for creating a session](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/). Similar to GCP, AWS uses an environment variable to find the default creds. I expect we will end up adding more query parameters here to override more `aws.Config` fields.

Reply:
I updated the comment to clarify and reference the [AWS documentation for creating a session](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/). Similar to GCP, it uses an environment variable to find the default credentials. I expect we will end up adding support for more query parameters here to override other `aws.Config` fields.